src/clj/rems/db/licenses.clj

```diff
@@ -61,3 +62,12 @@
        (format-licenses)
        (localize-licenses)
        (distinct-by :id)))
+
+(defn join-resource-licenses [x]
+  (assoc x :licenses (get-resource-licenses (:id x))))
+
+(defn join-catalogue-item-licenses [item]
+  (assoc item
+         :licenses
+         (get-licenses {:wfid (:wfid item)
+                        :items [(:id item)]})))
```

Comment:
the name might be misleading since they are actually resource licenses that are just looked up using catalogue items

Reply:
I think this confusion is already in the original code then. And we should check this up when adding workflow licenses and possibly other kinds of licenses.